src/test/java/org/openrewrite/java/testing/junit5/AssertToAssertionsTest.java

```diff
@@ -462,4 +462,35 @@ void test() {
           )
         );
     }
+
+    // edge case for deprecated use of assertEquals
+    // https://junit.org/junit4/javadoc/4.13/org/junit/Assert.html#assertEquals(java.lang.Object%5B%5D,%20java.lang.Object%5B%5D)
+    @Issue("#383")
+    @Test
+    void assertEqualsWithArrayArgumentToAssertArrayEquals() {
+        //language=java
+        rewriteRun(
+          spec -> spec.typeValidationOptions(TypeValidation.none()),
```

Comment:
```suggestion
```
Ideally this is not necessary; in rare cases where we want to make changes despite missing types we add that to some tests, but it could potentially break recipe chaining, as the next recipe will then not have the correct types available.

Reply:
Oh okay, I just copy-pasted it from the previous example without knowing what it really does exactly. Thanks for the explanation.

Reply:
You're welcome; thought as much seeing as the one just above had the same; figured point it out with a bit of background. :)